plugins/github/api.ml

```diff
@@ -174,10 +220,10 @@ let v ~get_token account =
   let head_monitors = Repo_map.empty in
   let refs_monitors = Repo_map.empty in
   let token_lock = Lwt_mutex.create () in
-  { get_token; token_lock; token = no_token; head_monitors; refs_monitors; account }
+  { get_token; token_lock; token = no_token; head_monitors; refs_monitors; account}
```

Comment:
This is a very odd whitespace change. I think it should either be `{a; b}` or `{ a; b }`, not `{ a; b}`. Same in other places.